posthog/models/app_metrics/sql.py

```diff
@@ -35,7 +40,7 @@
 )
 ENGINE = {SHARDED_APP_METRICS_TABLE_ENGINE()}
 PARTITION BY toYYYYMM(timestamp)
-ORDER BY (team_id, plugin_config_id, job_id, category, toStartOfHour(timestamp), error_type, error_uuid)
+ORDER BY (team_id, plugin_config_id, job_id, category, {APP_METRICS_TIMESTAMP_TRUNCATION}, error_type, error_uuid)
```

Comment:
I agree with the note, but this is merely an order by clause, no aggregation is happening here.

The group by I see is below in the `QUERY_APP_METRICS_TIME_SERIES` query which groups by `date` and a variable `interval`. That interval is set here as far as I could find: https://github.com/PostHog/posthog/blob/770c597dc02803bb3e246b3e4d6c3e88ece0fe65/posthog/queries/app_metrics/app_metrics.py#L73, and could be either hour or day according to https://github.com/PostHog/posthog/blob/770c597dc02803bb3e246b3e4d6c3e88ece0fe65/posthog/queries/app_metrics/app_metrics.py#L113

Reply:
So, if anything, the note should be in `app_metrics.py`. Granted, that's not a strong guarantee of anything, but at least it will be closer to the place where the aggregation is defined.

Reply:
Aah, my bad, I did not spot the table engine. Yeah, I think that should block anything lower than an hour. This is fine, thank you for pointing it out :+1: